tests/system/test_access.py

```diff
@@ -259,7 +259,8 @@ def setUp(self):
     def config(self):
         cfg = super(TestAccessWithAuthorization, self).config()
         cfg.update({"secret_token": "1234", "api_key_enabled": True, "enable_rum": True,
-                    "kibana_enabled": "true", "kibana_host": self.get_kibana_url()})
+                    "kibana_enabled": "true", "kibana_host": self.get_kibana_url(),
+                    "instrumentation_enabled": "true"})
```

Comment:
Why did you enable instrumentation here (you are not checking anything related to it afaics).

Reply:
I started updating this suite, and then decided to create a new suite - just forgot to revert this, which I've done now. In case you're wondering: I wanted the test to fail without the changes to beater, but it would have passed on account of the secret_token set in this suite's config.